demo/OrderCloud.DemoWebApi/Controllers/DemoController.cs

```diff
@@ -41,24 +41,17 @@ public class DemoController : BaseController
 		[HttpPost("modelvalidation")]
 		public ExampleModel ModelValidation(ExampleModel model) => model;
 
-		//[Route("webhook"), OrderCloudWebhookAuth]
-		//public object HandleAddressSave([FromBody] WebhookPayloads.Addresses.Save<MyConfigData> payload) {
-		//	return new {
-		//		Action = "HandleAddressSave",
-		//		City = payload.Request.Body.City,
-		//		Foo = payload.ConfigData.Foo
-		//	};
-		//}
-
-		//[Route("webhook"), OrderCloudWebhookAuth]
-		//public object HandleGenericWebhook([FromBody] WebhookPayload payload) {
-		//	return new {
-		//		Action = "HandleGenericWebhook",
-		//		City = payload.Request.Body.City,
-		//		Foo = payload.ConfigData.Foo
-		//	};
-		//}
-	}
+        [Route("webhook"), OrderCloudWebhookAuth]
+        public object HandleAddressSave([FromBody] WebhookPayloads.Addresses.Save payload)
+        {
+            return new
+            {
+                Action = "HandleAddressSave",
+                City = payload.Request.Body.City,
+                Foo = payload.ConfigData.Foo
+            };
+        }
+    }
```

Comment:
Can we add some comments? Also, the main use case of using disambiguate webhooks is when you want one route to handle multiple payloads. I think it'd be worth demonstrating at least another route (same route, different payload)

Reply:
I might also suggest we create a new file WebhookContoller.cs. Then if we want to show someone how to do webhooks we can link them to it and there's nothing else to confuse them.